testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/security/security_ssl_certificates.t.js

```diff
@@ -0,0 +1,85 @@
+
+  var sslCertificateListCQ = '>>nx-coreui-sslcertificate-list',
+      pemLoadCQ = '>>nx-coreui-sslcertificate-add-from-pem button[action=load]',
+      pemFieldCQ = '>>nx-coreui-sslcertificate-add-from-pem #pem',
+      addToTrustStoreButtonCQ = '>>nx-coreui-sslcertificate-details-form button[action=add][disabled=false]';
+
+  t.describe('SSL Certificates', function(t) {
+    t.it('An admin can load a certificate by pasting a PEM', function(t) {
+
+      t.chain(
+          t.openPageAsAdmin('admin/security/sslcertificates'),
+          {waitForCQVisible: sslCertificateListCQ},
+          {click: 'nx-coreui-sslcertificate-list button[text=Load certificate] => .x-btn-button'},
+          {click: '>>menuitem[action=newfrompem]'},
+          {waitForCQVisible: pemLoadCQ},
+          {
+            type: [
+              '-----BEGIN CERTIFICATE-----',
+              'MIIEBzCCAu+gAwIBAgIJAKfcFT5TCwrIMA0GCSqGSIb3DQEBCwUAMIGZMQswCQYD',
+              'VQQGEwJVUzERMA8GA1UECAwITWFyeWxhbmQxDzANBgNVBAcMBkZ1bHRvbjERMA8G',
+              'A1UECgwIU29uYXR5cGUxDDAKBgNVBAsMA0RldjEOMAwGA1UEAwwFbngtZnQxNTAz',
+              'BgkqhkiG9w0BCQEWJglzb25hdHlwZS1uZXh1cy1kZXYtZ3JvdXBAc29uYXR5cGUu',
+              'Y29tMB4XDTE1MDcwNzA0NTA1NloXDTI1MDcwNDA0NTA1NlowgZkxCzAJBgNVBAYT',
+              'AlVTMREwDwYDVQQIDAhNYXJ5bGFuZDEPMA0GA1UEBwwGRnVsdG9uMREwDwYDVQQK',
+              'DAhTb25hdHlwZTEMMAoGA1UECwwDRGV2MQ4wDAYDVQQDDAVueC1mdDE1MDMGCSqG',
+              'SIb3DQEJARYmCXNvbmF0eXBlLW5leHVzLWRldi1ncm91cEBzb25hdHlwZS5jb20w',
+              'ggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQClJk6Tcv8c2jQgsqIMns5c',
+              '5X+ZJWzVe5o+TTwDXOtdupK7MUt/vh7Q7AeThytXzjaWZDuZDf6m85OJvRTnmmDI',
+              'hMshU/TCek4GMXgcnDfRJFMXqlOd8fArn4lR+rHr6CsKt/P6S8A5B6hx24a6SL+p',
+              'a/W2MXInoerk67/syxecdDRew3K56qJ0yonDPNXrhWr5jsu0FERXGVYTreTi++Oz',
+              'qV/Of/mV/1IxefoZPsTLmkcrmekiS2jNcoZnwr312gySKl/v/jqAybjPsj7AYBEl',
+              '0/yjEnWLFjfTZCCixbQsOLGJBGraQnxpnO+G1OtSAuv7A0kd/MTypMkHNGlNXN09',
+              'AgMBAAGjUDBOMB0GA1UdDgQWBBQ3yInDrwNCBKN+Pn41+1M9o2YufDAfBgNVHSME',
+              'GDAWgBQ3yInDrwNCBKN+Pn41+1M9o2YufDAMBgNVHRMEBTADAQH/MA0GCSqGSIb3',
+              'DQEBCwUAA4IBAQBZa4WqQr74VTysIlZlaHw833mNQw238sgaOSG/RsTew3zR9d8t',
+              'jsOGgSCAGM4iTyAtLaEweMkDAPvXEL5B06KwULNDjoFrtTnN9rA2xzGzlDreLZfh',
+              'VHznzXgtzAXGF5CeG4q9p7U99m4bYpn9eYBfyDSo8ZKYvA79PNqAkVPfVlaLt46h',
+              'zWUSiCU8C0Uilam7UJHLI3I3hNIduKFrOSABDlWO4ZjHQ/XxMBKCPqDadeTTGY3P',
+              '/uW7JeOnXxWD4VVAuvHJFOGmaPaMoepQh1W4ueAy2GT1D45+PFeXQ8Yd0OO10c+n',
+              'wkOCAIGTk0wExqk/I/ytX9QrXdklowikn5se',
+              '-----END CERTIFICATE-----'
+            ].join('\n'), target: pemFieldCQ
+          },
+          {click: pemLoadCQ},
+          function(next) {
+            t.waitForAnimations(next);
+          },
+          {waitForCQVisible: addToTrustStoreButtonCQ, desc: 'button is enabled to add to trust store'},
```

Comment:
I'd also suggest we refrain from commenting code in favor of http://bryntum.com/docs/siesta/#!/api/Siesta.Test.Action .desc property, which shows up nicely in the test output and making it self-documenting